### ISSUE TITLE
[CEP-9] Permission Editor - Allow Admins and Sysadmins to edit (currently only sysadmins allowed)

### ISSUE THREAD

**Issue by csgruenebe** [codeclou/customfield-editor-plugin#9]
# Problem Currently no error message is displayed when user is not sysadmin

in Permission Editor

`/rest/jiracustomfieldeditorplugin/1.2/gui/searchUser?query=tim` returns 403

```
{
"statusCode": 403,
"subCode": 403001,
"message": "Permission Denied - you do not have sufficient rights for this operation."
}
```
# Solution
- Change to admin not only sysadmin
- display an error message once 403 is returned! that says "You have to be admin"

**Comment by clouless** [comment]
Fixed and part of next release.

**Comment by clouless** [comment]
Implemented with v1.3.0.
You can now download version 1.3.0 from the [Atlassian marketplace](https://marketplace.atlassian.com/plugins/jiracustomfieldeditorplugin/server/overview). 
See [Release Notes](https://codeclou.io/customfield-editor-plugin/1.3.0/release-notes/).
